ssv/spectest/tests/valcheck/valcheckattestations/slashable.go

```diff
@@ -49,7 +49,7 @@ func Slashable() tests.SpecTest {
 	return &valcheck.SpecTest{
 		Name:       "attestation value check slashable",
 		Network:    types.BeaconTestNetwork,
-		BeaconRole: types.BNRoleAttester,
+		RunnerRole: types.RoleCommittee,
 		Input:      input,
 		AnyError:   true,
```

Comment:
can you investigate why this test passes if we don't use BeaconVote?

what's going on here exactly?

Reply:
It works because there is `AnyError:   true,`, basically yes, the input has to be changed